crates/runtime/tests/models/hf.rs

```diff
@@ -297,7 +297,7 @@ mod search {
                         body: json!({
                             "text": "friends",
                             "datasets": ["catalog_page_with_chunking"],
-                            "additional_columns": ["cp_department", "cp_description"],
+                            "additional_columns": ["cp_catalog_page_sk", "cp_department", "cp_description"],
```

Comment:
pk?

Reply:
it is actually the TPCS naming, confirm it is a `cp_catalog_page_sk` / correct